aeron-system-tests/src/test/java/io/aeron/TaggedFlowControlSystemTest.java

```diff
@@ -249,12 +249,13 @@ void shouldRemoveDeadReceiver()
 
         launch();
 
-        subscriptionA = clientA.addSubscription(MULTICAST_URI + "|gtag=123", STREAM_ID);
-        subscriptionB = clientB.addSubscription(MULTICAST_URI + "|gtag=123", STREAM_ID);
         publication = clientA.addPublication(MULTICAST_URI + "|fc=tagged,g:123,t:1s", STREAM_ID);
 
+        subscriptionA = clientA.addSubscription(MULTICAST_URI + "|gtag=123", STREAM_ID);
```

Comment:
why the change in order?

Reply:
This gives the most chance of both receivers being included in flow control.

Reply:
I've been thinking about the potential of surfacing a num receivers counter for a publication so that we can see whether a receiver has been included in flow control before proceeding.